tests/unit/s2n_server_key_share_extension_test.c

```diff
@@ -280,6 +281,11 @@ int main(int argc, char **argv)
 
                 EXPECT_NOT_NULL(client_conn = s2n_connection_new(S2N_CLIENT));
 
+                /* Fake sending the key share extension */
+                s2n_extension_type_id key_share_id;
+                s2n_extension_supported_iana_value_to_id(s2n_server_key_share_extension.iana_value, &key_share_id);
```

Comment:
Even in tests, make sure you init. We've had failures with super old compilers before.
```suggestion
                s2n_extension_type_id key_share_id = 0;
                s2n_extension_supported_iana_value_to_id(s2n_server_key_share_extension.iana_value, &key_share_id);
```

Reply:
Fixed in the testlib functions in 0149747